add optional flag to follow the compilation output window (#70)
ej-shafran/compile-mode.nvim: doc/compile-mode.txt

    When `true`, the compilation buffer will be hidden, meaning *'buflisted'*
    will be `false` for it.

> follow_output_window                       *compile-mode.follow_output_window*

Reviewer: Maybe we can rename this option to `focus_compilation_buffer` or `focus_compilation_window`?

>     When `true`, after the compilation, compilation window will be set as the 
>     current window, meaning your cursor will jump to the compilation window.

Reviewer: ```suggestion
    When `true`, compiling will also set the compilation window as the
    current window.
```
